docs/docs/releases/0.2.0.md

```diff
@@ -456,23 +457,23 @@ The following PRs represent work done to reduce technical debt, update dependenc
 [#4152](https://github.com/mathesar-foundation/mathesar/pull/4152 "Release 0.2.0")
 [#4170](https://github.com/mathesar-foundation/mathesar/pull/4170 "Rpc performance")
 
-## Upgrading to 0.2.0 {:#upgrading}
-
-Upgrading to version 0.2.0 will:
+## Recover databases from a previous installation {:#recover}
```

Comment:
This title is inaccurate, and also scary for users. Maybe "reconnect"

Reply:
I've updated the PR to use: "Migrating to 0.2.0 from previous versions".